pkg/sql/authorization.go

```diff
@@ -138,11 +138,9 @@ func (p *planner) CheckPrivilege(
 	return p.CheckPrivilegeForUser(ctx, descriptor, privilege, p.User())
 }
 
-// IsOwner returns if the role has ownership on the descriptor.
-func IsOwner(desc catalog.Descriptor, role string) bool {
+func getOwnerOfDesc(desc catalog.Descriptor) string {
```

Comment:
wondering if this should be a property of Descriptor. Or named `descriptorOwner(desc)`.
this is a very minor nit and i don't mind either way - i just know [`Get` is kind of a frowned word in Go](https://golang.org/doc/effective_go.html#Getters).

Reply:
This is a good point, but I think should be done when we push out this authorization stuff from `sql`.